preference/java/springboot/pom.xml

```diff
@@ -44,7 +44,7 @@
         </dependency>
         <dependency>
             <groupId>com.uber.jaeger</groupId>
-            <artifactId>jaeger-core</artifactId>
+            <artifactId>jaeger-tracerresolver</artifactId>
```

Comment:
I am -1 on this one. 

I would rather see start using https://github.com/opentracing-contrib/java-spring-cloud/tree/master/opentracing-spring-cloud-starter-jaeger which uses spring native configuration for Jaeger tracer

Reply:
This is already being used by another module. For this PR, I'd try to keep it simple, as it's already bigger than I wanted.

Reply:
I would prefer to not change any dependencies as it would required to update the image on Katacoda at https://learn.openshift.com/servicemesh